src/compiler/commandLineParser.ts

```diff
@@ -1355,7 +1355,12 @@ namespace ts {
             description: Diagnostics.Control_what_method_is_used_to_detect_module_format_JS_files,
             category: Diagnostics.Language_and_Environment,
             defaultValueDescription: Diagnostics.auto_Colon_Treat_files_with_imports_exports_import_meta_jsx_with_jsx_Colon_react_jsx_or_esm_format_with_module_Colon_node16_as_modules,
-        }
+        },
+        {

```

Comment:
do we need to add it to `CompilerOptions` too ?

Reply:
It's not read anywhere, so I don't think there's any need to. It worked fine when I tested locally.